src/network/netdev/netdev-gperf.gperf

```diff
@@ -235,5 +235,7 @@ Batadv.BridgeLoopAvoidance,               config_parse_bool,
 Batadv.DistributedArpTable,               config_parse_bool,                         0,                             offsetof(Batadv, distributed_arp_table)
 Batadv.Fragmentation,                     config_parse_bool,                         0,                             offsetof(Batadv, fragmentation)
 Batadv.GatewayMode,                       config_parse_batadv_gateway_mode,          0,                             offsetof(Batadv, gateway_mode)
+Batadv.GatewayBandwithDown,               config_parse_uint32,                       0,                             offsetof(Batadv, gateway_bandwidth_down)
+Batadv.GatewayBandwithUp,                 config_parse_uint32,                       0,                             offsetof(Batadv, gateway_bandwidth_up)
```

Comment:
This should use `config_parse_iec_size` or similar. Or if none of the existing helpers are appropriate, please add a new one that wraps `parse_size`.